src/main/java/com/google/sps/data/BasicGameData.java

```diff
@@ -21,10 +21,10 @@
 * Represents only the info that is needed about a game from the main page.
 */
 public class BasicGameData {
+    MockDatastoreManager datastoreManager = new MockDatastoreManager();
     //TODO(ldchen): delete all of the mocking once datastore is working.
     private int getRandomIntegerBetween(int left, int right) {
```

Comment:
What's left and right?

Reply:
The lower and upper bounds for the integer this function will return. This function is documented somewhere else, but it will only be here temporarily for mocking purposes.